ketcher-autotests/tests/Templates/Template-Manipulations/Template-manipulations.spec.ts

```diff
@@ -0,0 +1,324 @@
+    Description: Look at the bottom of the application.
+    Choose any template.
+    */
+    await takeBottomToolbarScreenshot(page);
+  });
+});
+
+test.describe('Template Manupulations', () => {
+  test.beforeEach(async ({ page }) => {
+    await waitForPageInit(page);
+  });
+
+  test.afterEach(async ({ page }) => {
+    await takeEditorScreenshot(page);
+  });
+
+  test('1-2) Fuse atom-to-atom', async ({ page }) => {
+    /*
+    Test case: EPMLSOPKET-1674
+    Description: Create a structure from the template. 
+    Choose any element from the left panel or Periodic Table and click on any atom of the created structure.
+    */
+    const anyAtom = 0;
+    await drawBenzeneRing(page);
+    await selectAtomInToolbar(AtomButton.Oxygen, page);
+    await clickOnAtom(page, 'C', anyAtom);
+  });
+
+  test('3) Fuse atom-to-atom: drag atom slightly', async ({ page }) => {
+    /*
+    Test case: EPMLSOPKET-1674
+    Description: Create a structure from the template. 
+    Put the cursor on any other structure atom, click, and drag slightly.
+    */
+    const anyAtom = 0;
+    const x = 200;
+    const y = 200;
+    await drawBenzeneRing(page);
+    await moveOnAtom(page, 'C', anyAtom);
+    await dragMouseTo(x, y, page);
+    await resetCurrentTool(page);
+  });
+
+  test('5) Fuse atom-to-atom: click and drug atom to fuse atom-to-atom', async ({
+    page,
+  }) => {
+    /*
+    Test case: EPMLSOPKET-1674
+    Description: Put the cursor on any other structure atom, press, and drag. 
+    Release the cursor when the distance from the cursor to the selected atom is more than the bond length. 
+    */
+    await drawBenzeneRing(page);
+    const anyAtom = 0;
+    const x = 600;
+    const y = 600;
+    await drawBenzeneRing(page);
+    await moveOnAtom(page, 'C', anyAtom);
+    await dragMouseTo(x, y, page);
+    await resetCurrentTool(page);
+  });
+
+  test('Fuse atom-to-atom: click and drug atom to extand bonds', async ({
+    page,
+  }) => {
+    /*
+    Test case: EPMLSOPKET-1674
+    Description: Create a structure from the template. 
+    */
+    await drawBenzeneRing(page);
+    const anyAtom = 0;
+    const x = 700;
+    const y = 600;
+    await drawBenzeneRing(page);
+    await moveOnAtom(page, 'C', anyAtom);
+    await dragMouseTo(x, y, page);
+    await resetCurrentTool(page);
+  });
+
+  test('Fuse bond-to-bond', async ({ page }) => {
+    /*
+    Test case: 1676
+    Description:
+    Choose any template and click any bond of the created structure.
+    With the template select any bond of the created structure, hold and drag "left-right".
+    */
+    const shift = 10;
+    await selectLeftPanelButton(LeftPanelButton.SingleBond, page);
+    await clickInTheMiddleOfTheScreen(page);
+    await takeEditorScreenshot(page);
+    await page.keyboard.press('Control+a');
+    const { x: rotationHandleX, y: rotationHandleY } =
+      await getRotationHandleCoordinates(page);
+    await page.mouse.move(rotationHandleX, rotationHandleY);
```

Comment:
it's better to use dragMouseTo function - it would be easier to fix if needed

Reply:
done